backend/app/controllers/campaigns_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class CampaignsController < ApplicationController
```

Comment:
Will image upload be handled in this PR?

Reply:
Nope, just wanted to get this base PR asap since it has the views for the other models as well. index, admin_index, show and admin_show will work but the rest are not guaranteed to work.